src/view/viewTabs.component.tsx

```diff
@@ -99,7 +104,35 @@ function ViewTabs() {
     <Routes location={location}>
       <Route
         path={paths.home}
-        element={<Link to={'/catalogue'}>Browse the catalogue</Link>}
+        element={
+          <HomePage
+            logo={DGLogo}
+            backgroundImage={BackgroundImage}
+            greenSwirl1Image={GreenSwirl1Image}
+            greenSwirl2Image={GreenSwirl2Image}
+            decal1Image={Decal1Image}
+            decal2Image={Decal2Image}
+            decal2DarkImage={Decal2DarkImage}
+            decal2DarkHCImage={Decal2DarkHCImage}
+            facilityImage={FacilityImage}
+          />
+        }
+      />
```

Comment:
Could this be put into a a separate component in homePage.component, perhaps the current should be BaseHomePage or something a bit like the TranslatedHomepage in datagateway, and then this just HomePage so we don't have to import all of the images in the viewTabs component.